packages/node/src/indexer/api.service.ts

```diff
@@ -103,6 +113,10 @@ export class ApiService
     return this;
   }
 
+  get kyveApi(): KyveApi {
```

Comment:
This shouldn't be exposed, the goal should be to have `fetchBlocks` be called and if kyve is available try that otherwise use RPC